### ISSUE TITLE
Failed to initialize Insight Passthrough. (in Unity Playmode)

### ISSUE THREAD

**Issue by hyomin0804** [LocalJoost/MRTKQuestSeeThrough#3]
Hi! I've been enjoying your informative blogs and codes you've been sharing. I think your latest project, connecting MRTK3 with Meta Quest's Passthrough in an OpenXR Platform, is quite remarkable. However, even though I cloned the code and used it as is, I encountered a "Failed to initialize Insight Passthrough" error in play mode. Does this project not support play mode and can only be verified through building? Thanks as always for your help.😊

![스크린샷 2024-04-26 115433](https://github.com/LocalJoost/MRTKQuestSeeThrough/assets/87791247/734341c7-2854-431a-8840-57450c3bcfea)

**Comment by LocalJoost** [comment]
TBH, I never tried. Have you deployed it, and does it work on your Quest?

**Comment by hyomin0804** [comment]
I couldn't build the apk file. I am currently testing with the Quest 3 with Unity Editor version 2022.3.18f1. I get an error when I build the project for Android. Could you tell me which platform you choose? I think there are two possible reasons for the issue:

1. The Meta integration package installed(ver.47) in the current project is an old version that may not support Quest 3
2. There might be a mistake in the build settings..

1) I changed build platform to Android
![image](https://github.com/LocalJoost/MRTKQuestSeeThrough/assets/87791247/4766c7fe-7366-44e5-8ea8-af0edad316b9)

2) And fixed all project validation error

3) I faced the bunch of build errors.😂
![image](https://github.com/LocalJoost/MRTKQuestSeeThrough/assets/87791247/6839d16a-162e-489e-87d3-5b9c81287021)

**Comment by LocalJoost** [comment]
Yeah, I didn't test with newer Unity versions and yes, it probably uses old packages. As you may have noticed, the project is a bit older than the Oculus Quest 3. I could run the package, but I also got build errors, although different than yours - I get a Gradle error. I am sorry, I don't have solution ATM. I am currently building a new app that should run on Quest as well, so I will probably need to have a look into this subject, and will most likely publish new samples. Later.

**Comment by hyomin0804** [comment]
I found a way to connect MRTK3 with Quest 3 passthrough using "OpenXR Meta"😊, but it only works in build mode and not in play mode...

**Comment by hyomin0804** [comment]
Thanks for your comment!

**Comment by PavelKhabusov** [comment]
Same problem with v68 on Unity 2023.2.20f1

**Comment by LocalJoost** [comment]
Have you tried building the app and deploy it?
